lib/src/coap_config.dart

```diff
@@ -27,16 +25,6 @@ abstract class DefaultCoapConfig {
   /// The version of the CoAP protocol.
   String get version => 'RFC7252';
```

Comment:
Are we using this now? If we assume that UDP is RFC7252 and TCP is RFC8323 we could infer the encoder/decoder to use from the URI. Not sure if the user would need to specify a `version` then?

Reply:
I think it is more of a leftover from the time when there were multiple draft versions the user could choose from. In practice, I think we can omit the version field, as I suppose that CoAP for UDP will stay at version 1 for the foreseeable future, while CoAP for TCP and WebSockets don't define the field in their message formats at all.

Reply:
Initially, I already had it removed but then I was a bit unsure whether we should keep it. But I would also be in favor of dropping it :)

Reply:
I think it could be fine to remove it too, it could always be re-added later if multiple RFC's gain traction for the same network protocol